Obvious Fix: Fix application.yaml file name typo in config source docs

SohamJ opened the pull request:
Obvious Fix: Fix application.yaml file name typo in config source docs

geoand commented:
Thanks for the PR!

Since the `README.adoc` gets auto-generated, could you please update the PR to only contain the other change?

geoand commented:
Also, if you could please rebase onto the latest master (where I fixed an issue with code formatting configuration for the integration tests), that would be great

SohamJ commented:
> Also, if you could please rebase onto the latest master (where I fixed an issue with code formatting configuration for the integration tests), that would be great

Thanks for checking. I rebased the PR.
